source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,140 @@
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* add-ons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+They also include a compressed file for each installed add-on containing the add-on's version, config settings, and other data if any.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not.
+
+> Warning - Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+
+5. The only progress indicator at the moment is via the logs at the Supervisor > System page in the UI (or `ha su logs` in the CLI).
+6. To update the snapshots page in order to see current snapshots, use the reload icon in the upper right of the page.
+
+### Restoring a Snapshot on a new install
+
+Restoring a snapshot on a fresh install of Home Assistant OS requires that you first upload the snapshot to the backup directory on the new instance. There is no way to upload snapshots from the snapshot dashboard at this time so the first thing to do is install and configure an add-on of your preference in order to provide access to the `backup` directory. Samba, File Editor, or VSC are all good choices for this.
+
+After uploading to the `backup` directory, visit the snapshot page in the UI and reload the page using the icon in the upper right of the page to make the uploaded files visible. From there you can select your snapshot and then choose any or all parts of that snapshot to restore.
+
+> Note: Before restoring third-party add-ons, add their repositories to the add-on store page first! These are any add-ons which are not in the official repo or the community add-on repo. Nothing bad will happen, but those add-ons won't get restored. You can always add the repos and then go back and do a partial restore of the missing add-ons.
+
+### How long do snapshots take?
+
+The length of time it takes to create or restore snapshots will depend on how much you have to compress or decompress. A large database file or a collection of camera media captured by MotionEye will take longer to decompress (the default database is the sqllite home-assistant_v2.db in the config directory).
+
+See the [`recorder`](https://www.home-assistant.io/components/recorder/) integration page for ways to keep your database data down to a size that won't cause issues. Note the keep days, purge interval, and include/exclude options.
+
+When the restore is complete, you will lose connection to the UI and it will return just as though you had restarted Home Assistant. It can be a good idea to do an additional full reboot as well especially if add-ons appear to be missing from the restore. You can reboot from the Supervisor > System > Host card in the UI.
+
+### Creating snapshots using the cli
```

Comment:
Should we even mention this in common tasks, or should we just promote the usage of the CLI in general?

Reply:
This is part of the common task of making snapshots, but one of the provided methods is how I as seeing it.